watchmal/dataset/cnn/cnn_dataset.py

```diff
@@ -11,9 +11,10 @@
 
 # WatChMaL imports
 from watchmal.dataset.h5_dataset import H5Dataset
-from watchmal.dataset.cnn_mpmt import transformations
 import watchmal.dataset.data_utils as du
 
+n_channels=2
```

Comment:
I think global variables like this should be avoided. Ideally there should be a config parameter to choose which channels to use (e.g. see how the PointNet class does it), or if you always want to use both then I don't think it needs to be a variable at all.

Reply:
I removed this global variable and added the option to train on times, charges, or both.